Src/Generic/Extensions/DeviceKit+RoundedCorners.swift

```diff
@@ -10,16 +10,16 @@ import DeviceKit
 import Foundation
 
 extension Device {
-    static var hasRoundedCorners: Bool = Device().isOneOf([
+    static var hasRoundedCorners: Bool = Device.current.isOneOf([
```

Comment:
DeviceKit for Swift 5 has the following: "New .allDevicesWithRoundedDisplayCorners and .hasRoundedDisplayCorners values to check if a device has rounded display corners. (eg. iPhone Xs and iPad Pro (3rd generation))"